app/src/main/res/values/strings.xml

```diff
@@ -13,61 +13,70 @@
     <!-- Theme Preferences -->
     <string name="theme_title">Application Theme</string>
     <string name="theme_header">Theme</string>
-
+    <!-- Notifications -->
```

Comment:
A comment is only visible for the string right under it in weblate. However, I don't think comments are necessary for all strings, a lot of strings are easy to understand what they are for.